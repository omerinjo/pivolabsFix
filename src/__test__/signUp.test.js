import configureStore from 'redux-mock-store'

import { renderComponent } from './index'
import Home from '../screens/signup/signApp';

const initialState = {};

const mockStore = configureStore();
let store;

beforeEach(() => {
    //creates the store with any initial state or middleware needed  
    store = mockStore(initialState)
})


describe('<Home> rendering', () => {
    test('render correctly', () => {
        const component = renderComponent(<Home store={store} />)
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})