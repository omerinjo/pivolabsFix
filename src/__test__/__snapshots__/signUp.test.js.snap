// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home> rendering render correctly 1`] = `
<div
  className="container"
>
  <div
    className=""
  >
    <nav
      className="navbar-container navbar navbar-expand-lg navbar-light bg-light"
      data-test="guestNavigation"
    >
      <a
        className="nav-link"
        href="/"
        onClick={[Function]}
      >
        Home 
      </a>
      <div
        className="mr-auto navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          href="/login"
          onClick={[Function]}
        >
          Login 
        </a>
        <a
          className="nav-link"
          href="/standing"
          onClick={[Function]}
        >
          Standing 
        </a>
      </div>
    </nav>
  </div>
  <form
    className=""
    onSubmit={[Function]}
  >
    <div
      className="form-group"
    >
      <label
        className="form-label"
        htmlFor="validationCustom01"
      >
        First Name
      </label>
      <input
        className="form-control"
        id="validationCustom01"
        name="first_name"
        onChange={[Function]}
        placeholder="Enter first name"
        type="text"
      />
    </div>
    <div
      className="form-group"
    >
      <label
        className="form-label"
        htmlFor="validationCustom02"
      >
        Last Name
      </label>
      <input
        className="form-control"
        id="validationCustom02"
        name="last_name"
        onChange={[Function]}
        placeholder="Enter last name"
        type="text"
      />
    </div>
    <div
      className="form-group"
    >
      <label
        className="form-label"
        htmlFor="formGroupEmail"
      >
        Email address
      </label>
      <input
        className="form-control"
        id="formGroupEmail"
        name="email"
        onChange={[Function]}
        placeholder="Enter email"
        type="email"
      />
    </div>
    <div
      className="form-group"
    >
      <label
        className="form-label"
        htmlFor="formGroupPassword"
      >
        Password
      </label>
      <input
        className="form-control"
        id="formGroupPassword"
        name="password"
        onChange={[Function]}
        placeholder="Password"
        type="password"
      />
    </div>
    <button
      className="btn btn-primary"
      disabled={false}
      type="submit"
    >
      Submit
    </button>
  </form>
</div>
`;
